Taskfile.yml

```diff
@@ -27,16 +27,16 @@ vars:
       "version": {"ref": "pizza"}
     }
   #
-  GOLANGCI_VERSION: v1.52.2
-  TPARSE_VERSION: v0.12.1
+  GOLANGCI_VERSION: v1.53.3
+  GOTESTSUM_VERSION: v1.10.1
 
 tasks:
 
   install:deps:
     desc: Install tool dependencies.
     cmds:
       - go install github.com/golangci/golangci-lint/cmd/golangci-lint@{{.GOLANGCI_VERSION}}
-      - go install github.com/mfridman/tparse@{{.TPARSE_VERSION}}
+      - go install gotest.tools/gotestsum@{{.GOTESTSUM_VERSION}}
```

Comment:
Can you write a few words on the reasoning here?

Reply:
The commit message for #79b9ce52c36 has an explanation. One problem is that tparse is used as a pipe, so if the previous stage (go test) fails, sometimes tparse cannot detect it (same problem with shell and -o pipefail). On the other hand gotestsum is used as a wrapper, so it can detect any failure of the child.